runner_manager/models/runner_group.py

```diff
@@ -353,10 +365,17 @@ def delete(
         """
         group: RunnerGroup = cls.get(pk)
         runners: List[Runner] = group.get_runners()
+        """for runner in runners:
```

Comment:
Hum what is that?

Reply:
just some old lines of code that i forgot to remove :)